smithy-cli/src/main/java/software/amazon/smithy/cli/commands/Upgrade1to2Command.java

```diff
@@ -321,6 +324,16 @@ public Void stringShape(StringShape shape) {
             return null;
         }
 
+        @Override
+        public Void listShape(ListShape shape) {
+            getDefault(shape);
+            if (shape instanceof SetShape) {
+                writer.updateLine(shape.getSourceLocation().getLine(),
+                    line -> line.replace("set", "@uniqueItems\nlist"));
```

Comment:
Using replace is brittle as some other occurrences of the characters "set" will be also replace, e.g.,

```
jshell> "set Fooset { // this set is awesome".replace("set", "@uniqueItems\nlist")
$1 ==> "@uniqueItems\nlist Foo@uniqueItems\nlist { // this @uniqueItems\nlist is awesome"
```

Using replace first might help but that might still fail for some other cases, e.g.,

```
jshell> "/* this set is awesome */ set Foo {".replaceFirst("set", "@uniqueItems\nlist")
$2 ==> "/* this @uniqueItems\nlist is awesome */ set Foo {"
```

Reply:
The second one is wrong, we don't have block comments but the argument still stands, e.g.,

```
"@tags([\"set\"]) set Foo {".replaceFirst("set", "@uniqueItems\nlist")
$3 ==> "@tags([\"@uniqueItems\nlist\"]) set Foo {"
```